src/completer/model.ts

```diff
@@ -263,16 +261,23 @@ class CompleterModel implements CompleterWidget.IModel {
         results.push({
           raw: option,
           score: match.score,
-          child: StringExt.highlight(option, match.indices, h.mark)
+          text: StringExt.highlight(option, match.indices, this._mark).join('')
```

Comment:
`this._mark` *works* because you don't touch `this` in that function. But it's technically incorrect because the method will be called unbound. Better to make a free function in the private namespace to do that.